.circleci/config.yml

```diff
@@ -41,212 +41,62 @@ jobs:
   platform-1-12-10:
     machine:
       image: ubuntu-1604:201903-01
-      resource_class: large
+      resource_class: 2xlarge
     environment:
       KUBE_VERSION: v1.12.10
     steps:
       - helm-install:
-          astronomer-tags: "platform postgresql"
-  monitoring-1-12-10:
-    machine:
-      image: ubuntu-1604:201903-01
-    environment:
-      KUBE_VERSION: v1.12.10
-    steps:
-      - helm-install:
-          astronomer-tags: "monitoring postgresql"
-  logging-1-12-10:
-    machine:
-      image: ubuntu-1604:201903-01
-      resource_class: large
-    environment:
-      KUBE_VERSION: v1.12.10
-    steps:
-      - helm-install:
-          astronomer-tags: "logging"
-  kubed-1-12-10:
-    machine:
-      image: ubuntu-1604:201903-01
-    environment:
-      KUBE_VERSION: v1.12.10
-    steps:
-      - helm-install:
-          astronomer-tags: "kubed"
+          astronomer-tags: "platform postgresql monitoring logging kubed keda"
```

Comment:
nice!

Reply:
a little history - previously we were doing it separately because we did not have an option to run in bigger VMs.